packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -107,6 +110,33 @@ class PurchaseParam {
 
   /// The 'sandboxTesting' is only available on iOS, set it to `true` for testing in AppStore's sandbox environment. The default value is `false`.
   final bool sandboxTesting;
+
+  /// The 'changeSubscriptionParam' is only available on Android, for upgrading or
+  /// downgrading an existing subscription.
+  final ChangeSubscriptionParam changeSubscriptionParam;
+}
+
+/// The parameter object for upgrading or downgrading an existing subscription so that
+/// the user can only purchase one subscription in a group at a time. This object
+/// is only applicable for Android.
+///
+/// This object does not require on iOS since Apple provides a way to create a
```

Comment:
Can we explain more on iOS here? For example how exactly changing subscription plan can be done on iOS, what is required from a developer, or maybe nothing needs to be done by a developer? Maybe we can put a link here?

Reply:
Done